doc/whats_new/v0.20.rst

```diff
@@ -27,6 +27,13 @@ Changelog
   graph, which would add explicitly zeros on the diagonal even when already
   present. :issue:`12105` by `Tom Dupre la Tour`_.
 
+:mod:`sklearn.compose`
+......................
+
+- |Fix| Fixed a bug in :class:`compose.ColumnTransformer` where it failed
+  given columns of mixed types with a single transformer.
+  :issue:`11912` by :user:`Adrin Jalali <adrinjalali>`.
```

Comment:
This is only specific to passthrough, not any transformer, right? Otherwise looks good.

Reply:
No, this is relevant for any transformer (passthrough is converted to a FunctionTransformer), which outputs non-numerical values. Of course in scikit-learn itself, I assume you can only get this with a custom transformation like FunctionTransformer

Reply:
But the "with a single transformer", I don't fully understand (I don't think it matters).

Reply:
@jorisvandenbossche, I agree it's not the best description, but it's the best I could come up with in a single sentence. I'm open to suggestions.

Reply:
I would say "fixed an issue when stacking dataframes with sparse matrices" maybe? Because that's really what it boils down to, I think.

Reply:
that sounds indeed better. Maybe ".. stacking mixed-type DataFrames .." as it is only when you have a DataFrames that will coerce to object

Reply:
Although, "mixed type" is also not fully correct, as you will also get this with simply a dataframe that has only string columns. 
So maybe " .. stacking DataFrames with non-numerical columns .."